tests/manager/manager-ctor-009.phpt

```diff
@@ -0,0 +1,38 @@
+--TEST--
+MongoDB\Driver\Manager::__construct() allows disabling client persistence
+--FILE--
+<?php
+
+ini_set('mongodb.debug', 'stderr');
+new MongoDB\Driver\Manager();
+
+// Won't reuse first client instance due to different options
+new MongoDB\Driver\Manager(null, [], ['disableClientPersistence' => false]);
+// Will reuse the previously created client
+new MongoDB\Driver\Manager(null, [], ['disableClientPersistence' => false]);
+
+// Will create a new private client
+new MongoDB\Driver\Manager(null, [], ['disableClientPersistence' => true]);
+
+// Will create another new private client
+new MongoDB\Driver\Manager(null, [], ['disableClientPersistence' => true]);
+ini_set('mongodb.debug', '');
+
+?>
+===DONE===
+<?php exit(0); ?>
+--EXPECTF--
+%A
+[%s]     PHONGO: DEBUG   > Created client with hash: %s
+[%s]     PHONGO: DEBUG   > Persisted client with hash: %s
+[%s]     mongoc: DEBUG   > Not destroying persistent client for Manager%A
```

Comment:
Note that the log domain is `mongoc` here, despite this message originating in ext-mongodb. I'll have to investigate that separately.

Reply:
Looks like this has always been the case. You can fix it by adding this at the top of the file _after_ including headers (just in case it conflicts with something in the libmongoc/libbson headers):

```
#undef MONGOC_LOG_DOMAIN
#define MONGOC_LOG_DOMAIN "PHONGO"
```